content/en/flux/installation/_index.md

```diff
@@ -0,0 +1,308 @@
+
+With [Chocolatey](https://chocolatey.org/) for Windows:
+
+```powershell
+choco install flux
+```
+
+{{% /tab %}}
+{{< /tabpane >}}
+
+To configure your shell to load `flux` [bash completions](./cmd/flux_completion_bash.md) add to your profile:
+
+```sh
+. <(flux completion bash)
+```
+
+[`zsh`](./cmd/flux_completion_zsh.md), [`fish`](./cmd/flux_completion_fish.md),
+and [`powershell`](./cmd/flux_completion_powershell.md)
+are also supported with their own sub-commands.
+
+A container image with `kubectl` and `flux` is available on DockerHub and GitHub:
+
+* `docker.io/fluxcd/flux-cli:<version>`
+* `ghcr.io/fluxcd/flux-cli:<version>`
+
+## Bootstrap
+
+Using the `flux bootstrap` command you can install Flux on a
+Kubernetes cluster and configure it to manage itself from a Git
+repository.
+
+If the Flux components are present on the cluster, the bootstrap
+command will perform an upgrade if needed. The bootstrap is
+idempotent, it's safe to run the command as many times as you want.
+
+The Flux component images are published to DockerHub and GitHub Container Registry
+as [multi-arch container images](https://docs.docker.com/docker-for-mac/multi-arch/)
+with support for Linux `amd64`, `arm64` and `armv7` (e.g. 32bit Raspberry Pi)
+architectures.
+
+If your Git provider is **AWS CodeCommit**, **Azure DevOps**, **Bitbucket Server**, **GitHub** or **GitLab** please
+follow the specific bootstrap procedure:
+
+* [AWS CodeCommit](./bootstrap/aws-code-commit.md#flux-installation-for-aws-codecommit)
+* [Azure DevOps](./bootstrap/azure.md#flux-installation-for-azure-devops)
+* [Bitbucket Server and Data Center](./bootstrap/bitbucket.md#bitbucket-server-and-data-center)
+* [GitHub.com and GitHub Enterprise](./bootstrap/github.md#github-and-github-enterprise)
+* [GitLab.com and GitLab Enterprise](./bootstrap/gitlab.md#gitlab-and-gitlab-enterprise)
+
+## Bootstrap with Terraform
+
+The bootstrap procedure can be implemented with Terraform using the Flux provider published on
+[registry.terraform.io](https://registry.terraform.io/providers/fluxcd/flux).
+The provider offers a Terraform resource called
+[flux_bootstrap_git](https://registry.terraform.io/providers/fluxcd/flux/latest/docs/resources/bootstrap_git)
+that can be used to bootstrap Flux in the same way the Flux CLI does it.
+
+Example of Git HTTPS bootstrap:
+
+```hcl
+provider "flux" {
+  kubernetes = {
+    config_path = "~/.kube/config"
+  }
+  git = {
+    url  = var.gitlab_url
+    http = {
+      username = var.gitlab_user
+      password = var.gitlab_token
+    }
+  }
+}
+
+resource "flux_bootstrap_git" "this" {
+  path                   = "clusters/my-cluster"
+  network_policy         = true
+  kustomization_override = file("${path.module}/kustomization.yaml")
+}
+```
+
+For more details on how to use the Terraform provider
+please see the [Flux docs on registry.terraform.io](https://registry.terraform.io/providers/fluxcd/flux/latest/docs).
+
+## Customize Flux manifests
```

Comment:
This should be in the "Configuration" page.

Reply:
Noted. You will see it on the Configuration page when I create a PR for it.